### DISCUSSION TITLE
Istanza di staging (test)

### DISCUSSION THREAD

**Discussion by pdonadeo** [pdonadeo/e-uscito-joypad#22] [Announcements]
Ciao a tutti, come [preannunciato nella discussione sul frontend](https://github.com/pdonadeo/e-uscito-joypad/discussions/16#discussioncomment-2793493) ho messo on line una istanza di staging[^1] che essenzialmente contiene una preview del lavoro di @MarcoJul. Conto di tenerla molto più aggiornata della versione stabile.

Ci potete arrivare qui: https://staging.euscitojoypad.it/

Vi chiede un utente e una password: non è una vera sicurezza, serve solo per tenere lontano i non addetti ai lavori e i bot. Utente: **`joypad`**, l'altro pezzo della sicurezza è **`corrisaltaspara`**.

[^1]: per @zampale: si tratta di una versione dell'applicazione che contiene una preview di quel che verrà, ancora instabile.

**Comment by agofa** [comment]
Bene, grazie
Fateci sapere quando avrà senso fare test

**Reply by pdonadeo** [reply]
Direi che sta a @MarcoJul dare il via :)

**Reply by MarcoJul** [reply]
Preparerò ogni tanto un cartello su fondo giallo stile CD Project in cui annuncerò i vari rinvii della deadline...

E dire che il sito sul mio computer "gira sorprendentemente bene"😆

A parte gli scherzi, sono piano piano al lavoro e non vedo l'ora di darvi il via!

**Reply by agofa** [reply]
> A parte gli scherzi, sono piano piano al lavoro e non vedo l'ora di darvi il via!

Manca poco al completamento dei riascolti
Ci dici tu quando sarà possibile fare test?
Sto provando la ricerca e dà qualche risultato non corretto

**Comment by agofa** [comment]
Ho iniziato ad usarla. C'è to the moon in offerta su Steam e sono andato a sentire che ne hanno detto i 3 vati...
Direi che è già molto usabile anche con il mio 5.8".

Qualche cosa da sistemare nell'interfaccia ma direi nulla di drammatico. 

![Screenshot_20220629-134748_Firefox.jpg](https://user-images.githubusercontent.com/19308040/176429649-04a420db-3550-4f30-8ba0-5248c4f91a48.jpg)

**Reply by MarcoJul** [reply]
Ottimo, grazie per la segnalazione! 

Al momento ho usato come risoluzione minima il 375px, potrei sapere che specifico smartphone hai o se lo sai la misura del lato corto? 
Te lo chiedo per cercare di capire che problema possa essere.

Comunque appunto, appena vedete bug o malfunzionamenti fatemi sapere che vediamo di sistemare tutto.

Nel frattempo la situazione generale è andata un po' avanti rispetto allo staging, ora anche la visualizzazione desktop dovrebbe essere  quasi utilizzabile, domani correggo qualche bug e si può procedere con l'aggiornamento.

**Reply by MarcoJul** [reply]
Ah e aspettiamo feedback anche su To The Moon, a me era piaciuto parecchio!😉

**Reply by agofa** [reply]
> Al momento ho usato come risoluzione minima il 375px, potrei sapere che specifico smartphone hai o se lo sai la misura del lato corto?

Scusa ho postato un messaggio prima di leggere questo
Samsung SM-G960F, 5,8" usando Firefox

@MarcoJul scrivo qui oppure apro issue?

**Reply by agofa** [reply]
> Ah e aspettiamo feedback anche su To The Moon, a me era piaciuto parecchio!😉

Eh prima ho un trascririficio da mandare avanti ;-)

**Reply by MarcoJul** [reply]
> > Al momento ho usato come risoluzione minima il 375px, potrei sapere che specifico smartphone hai o se lo sai la misura del lato corto?
> 
> Scusa ho postato un messaggio prima di leggere questo Samsung SM-G960F, 5,8" usando Firefox
> 
> @MarcoJul scrivo qui oppure apro issue?

non saprei, per le segnalazioni feedback possiamo fare qua, o su "sezione frontend", o su issue, l'importante è che siano tutte nello stesso posto... chiediamo a @pdonadeo, il nostro padrone di casa, come ha in mente di organizzare.

**Reply by pdonadeo** [reply]
@agofa apri pure una issue qui: https://github.com/pdonadeo/e-uscito-joypad/issues

Poi nel caso la assegno a me o @MarcoJul.

**Reply by agofa** [reply]
> @agofa apri pure una issue qui

Però vi consiglio di non trattenere il respiro nell'attesa

**Reply by agofa** [reply]
> Al momento ho usato come risoluzione minima il 375px, potrei sapere che specifico smartphone hai o se lo sai la misura del lato corto?

@MarcoJul, dimentica questa segnalazione. Mi ero dimenticato che per altri test avevo aumentato la dimensione del font oltre ogni ragionevole limite...

**Reply by MarcoJul** [reply]
Ok ! La segnalazione è comunque sensata perché da li ho fatto un attimo di ricerca sui telefoni android e ho visto che anche dei Samsung Galaxy non cosi preistorici hanno una risoluzione lato corto di 360px (contro i 375px di minimo che ho considerato io).  

Non una grossa problematica ma la tengo a mente come verifica.

**Reply by agofa** [reply]
In ogni caso problema risolto con dimensione font ragionevole

**Comment by zampale** [comment]
❤️
Il 29 giu 2022, 13:51 +0200, Agostino Fanti ***@***.***>, ha scritto:
